Exercise6/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter the first edge: ");
+            var firstEdge = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Enter the second edge: ");
+            var secondEdge = Convert.ToInt32(Console.ReadLine());
+            Console.Write("Enter the third edge: ");
+            var thirdEdge = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Type: " + GetTypeOfTriangle(firstEdge, secondEdge, thirdEdge));
```

Comment:
You should say 
```C#
if(IsEdgesOfTriangle(....))
{
    Console.WriteLine("Type: " + GetTypeOfTriangle(firstEdge, secondEdge, thirdEdge));
    Console.WriteLine("Perimeter: " + CalculatePerimeter(firstEdge, secondEdge, thirdEdge));
    Console.WriteLine("Area: " + CalculateArea(firstEdge, secondEdge, thirdEdge));
}
else
{
    Console.WriteLine(...);
}
```

And remove call `IsEdgesOfTriangle` from `CalculatePerimeter` `GetTypeOfTriangle`

Reply:
Done